src/TypeScript/Analyzer/EslintBridgeAnalyzer.cs

```diff
@@ -133,6 +133,10 @@ private async Task EnsureEslintBridgeClientIsInitialized(CancellationToken cance
                     shouldInitLinter = false;
                 }
             }
+            catch (Exception ex)
```

Comment:
For some reason If we don't add this, we publish whole stack trace of timeout exception to the output window. But when we add this, errors are not written at all. 🤔